engine/build.gradle

```diff
@@ -156,6 +156,7 @@ dependencies {
 
     // Wildcard dependency to catch any libs provided with the project (remote repo preferred instead)
     implementation fileTree(dir: 'libs', include: '*.jar')
+    apiElements fileTree(dir: 'libs', include: '*.jar')
```

Comment:
`implementation` + `apiElements` = `api`

Reply:
and `api` at https://github.com/MovingBlocks/Terasology/pull/3990/files#diff-1d22d9382cea237552a9f1e8e4485485R118